docs/notebooks/first_tutorial.jl

```diff
@@ -429,6 +482,12 @@ let
 end
   ╠═╡ =#
 
+# ╔═╡ 100fed92-2d7f-44c3-8ad3-020638c05b5f
```

Comment:
Maybe also provide a solution?

Reply:
Yeah, though this is doable from the exported data, it requires some data mangling that is outside the scope of the tutorial, so I removed this exercise.